devicetypes/smartthings/zwave-basic-smoke-alarm.src/README.md

```diff
@@ -22,18 +23,62 @@ Works with:
 
 ## Device Health
 
-First Alert Smoke Detector (ZSMOKE) is a Z-wave sleepy device and checks in every 1 hour.
+First Alert Smoke Detector (ZSMOKE) and FireAngel Thermoptek ZST-630 Smoke Alarm/Detector are a Z-wave sleepy devices and checks in every 1 hour.
 Device-Watch allows 2 check-in misses from device plus some lag time. So Check-in interval = (2*60 + 2)mins = 122 mins.
 
 * __122min__ checkInterval
 
 ## Battery Specification
 
-Two AA 1.5V batteries are required.
-
+First Alert Smoke Detector (ZSMOKE) Two AA 1.5V batteries are required.
+FireAngel Thermoptek ZST-630 Smoke Alarm/Detector One CR2 battery required
 ## Troubleshooting
 
 If the device doesn't pair when trying from the SmartThings mobile app, it is possible that the device is out of range.
 Pairing needs to be tried again by placing the device closer to the hub.
 Instructions related to pairing, resetting and removing the device from SmartThings can be found in the following link:
 * [First Alert Smoke Detector (ZSMOKE) Troubleshooting Tips](https://support.smartthings.com/hc/en-us/articles/207150556-First-Alert-Smoke-Detector-ZSMOKE-)
+* [FireAngel Thermoptek ZST-630 Smoke Alarm/Detector Troubleshooting Tips]
+### To connect the FireAngel Thermoptek ZST-630 Smoke Alarm/Detector with the SmartThings Hub
+```
+Insert the batterry into Z-Wave module (provided separately).
+
+Then, in the SmartThings mobile app:
+
+Tap My Home 
+Under Things, tap Add a Thing at the bottom of your Things list
+The app will say Looking for devices�
```

Comment:
Not a big deal, but It's not great to be adding documentation for the mobile app to these comments. This UI regularly changes and we don't want a bunch of outdated comments all over the place. Just say "put the hub in Add Device mode" or something like that.

Reply:
Ok i will shorten the steps